src/main/java/com/sender/email/EmailService.java

```diff
@@ -0,0 +1,81 @@
+package com.sender.email;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.mail.SimpleMailMessage;
+import org.springframework.mail.javamail.JavaMailSender;
+import org.springframework.scheduling.TaskScheduler;
+import org.springframework.stereotype.Service;
+import javax.annotation.PostConstruct;
+import java.util.Date;
+import java.util.Scanner;
+
+@Service
+public class EmailService {
+    private Email email;
+
+    private TaskScheduler taskScheduler;
+
+    private JavaMailSender javaMailSender;
+
+    private SimpleMailMessage simpleMailMessage;
+
+
+    public class EmailSending implements Runnable {
+        @Override
+        public void run() {
+            simpleMailMessage.setTo(getEmail().getRecipient());
+            simpleMailMessage.setSubject(getEmail().getSubject());
+            simpleMailMessage.setText(getEmail().getBody());
+            javaMailSender.send(simpleMailMessage);
+            getEmail().setSentStatus(true);
+            getEmail().loadIntoFile();
+            System.out.println("Email has been sent successfully!");
+            System.exit(0);
+        }
+    }
+
+    @PostConstruct
+    public void postInit() {
```

Comment:
It would be nice to move this into entrypoint or in a separate class. Consider using [CommandLineRunner](https://www.baeldung.com/spring-boot-console-app)